dokang/utils.py

```diff
@@ -7,10 +7,16 @@
 import os.path
 
 from dokang import api
+from dokang import compat
 
 
 def get_harvester(fqn):
-    module_fqn, function_fqn = fqn.rsplit(b'.', 1)
+    module_fqn, function_fqn = fqn.rsplit('.', 1)
+
+    if compat.PY2 and not isinstance(module_fqn, bytes):
+        module_fqn = module_fqn.encode()
+        function_fqn = function_fqn.encode()
```

Comment:
Unless I missed the point, it does not like it is mandatory:

    $ python2
    Python 2.7.11 (default, Mar  3 2016, 11:00:04) 
    [GCC 5.3.0] on linux2
    Type "help", "copyright", "credits" or "license" for more information.
    Python shell history and tab completion are enabled.
    >>> m = __import__(u'os', fromlist=[u'listdir'])
    >>> getattr(m, u'listdir')
    <built-in function listdir>

I suppose that it is useful (or even required) if the module/function contains non-ASCII characters, but that is not likely to happen, is it? I'd rather not add `if compat.PY2` code if we can avoid it.